_assets/css/_sidebar.scss

```diff
@@ -249,7 +268,7 @@ $sidebar-nav-width: rem-calc(240);
 // Side Nav Detail Loop
 
 .site.sidebar .sidenav-detail {
-  padding-bottom: rem-calc(16);
+  padding-bottom: rem-calc(64);
```

Comment:
I am not sure how this works but there are a few values that are repeated. We could maybe add variables to improve readibility.

Reply:
Yeah, vars sounds good, we'd need to know what they represent though.